doc/output-validation.ipynb

```diff
@@ -0,0 +1,830 @@
+    "tags": []
+   },
+   "source": [
+    "# Imports"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "da70e609-6e4a-43e1-b4da-17c7572d6b36",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# Pre-requisites\n",
+    "import os\n",
+    "import numpy as np\n",
+    "import tables_io\n",
+    "import qp \n",
+    "import os\n",
+    "##import holoviews as hv\n",
+    "import pandas as pd\n",
+    "import h5py\n",
+    "\n",
+    "from matplotlib import pyplot as plt \n",
+    "from pathlib import Path\n",
+    "#hv.extension('bokeh')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d2c52d24-2e5e-4312-a0bd-11571d87974e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Rail modules\n",
+    "import rail\n",
+    "\n",
+    "from rail.core.data import TableHandle, QPHandle, Hdf5Handle, PqHandle\n",
+    "from rail.core.stage import RailStage\n",
+    "from rail.core.util_stages import ColumnMapper, TableConverter\n",
+    "from rail.core.utils import find_rail_file\n",
+    "\n",
+    "from rail.estimation.algos.naive_stack import NaiveStackSummarizer\n",
+    "from rail.estimation.algos.point_est_hist import PointEstHistSummarizer\n",
+    "\n",
+    "from rail.evaluation.dist_to_dist_evaluator import DistToDistEvaluator # pdfs true vs pdfs caldulada\n",
+    "from rail.evaluation.dist_to_point_evaluator import DistToPointEvaluator # zmode calculado vs pdfs true\n",
+    "from rail.evaluation.point_to_point_evaluator import PointToPointEvaluator # zmode true vs zmode calculado\n",
+    "from rail.evaluation.single_evaluator import SingleEvaluator\n",
+    "from rail.evaluation.evaluator import OldEvaluator\n",
+    "\n",
+    "from rail.evaluation.metrics.cdeloss import *\n",
+    "from rail.evaluation.metrics.pointestimates import PointSigmaIQR, PointBias, PointOutlierRate, PointSigmaMAD, PointStatsEz\n",
+    "\n",
+    "from utils import plot_pit_qq, ks_plot, read_pz_output, plot_pdfs, plot_old_valid, old_metrics_table #copied utils.py from rail/examples/evaluation_examples\n",
+    "\n",
+    "from qp.metrics.pit import PIT\n",
+    " \n",
+    "\n",
+    "%matplotlib inline\n",
+    "%reload_ext autoreload\n",
+    "%autoreload 2\n",
+    "\n",
+    "DS = RailStage.data_store\n",
+    "DS.__class__.allow_overwrite = True"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a4e447e3-254d-4394-9ed1-47580ea2bc01",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "# Reading the files\n",
+    "\n",
+    "Here we are going to load the files and have a look trough the output file of a pz-estimate to better undestand the structure of the output data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "408f6c3c-bdd6-4766-a023-dda32f18ce38",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "validation_set_output_path = 'validation_set_output.hdf5'\n",
```

Comment:
As I got the validation_set files via pzserver, using the commands I mentioned before, I had to modify these lines to work:
`validation_set_output_path = '<path-to-pz-compute-local-repo>/doc/validation_set_output.hdf5'`
`validation_set_path = '<path-to-pz-compute-local-repo>/doc/validation_set.hdf5'`

Reply:
added along with the pz-server block to import the files